d3ploy/no_inst.py

```diff
@@ -32,7 +32,8 @@ class NOInst(Institution):
         doc="A list of commodities that the institution will manage.",
         tooltip="List of commodities in the institution.",
         uilabel="Commodities",
-        uitype="oneOrMore"    
+        uitype="oneOrMore"
+    )
 
```

Comment:
What happened here? Was this always missing a closing parenthesis ?

Reply:
Yeah I pulled from ergs/develop and it was missing a parenthesis

Reply:
weird

Thanks for fixing it !